local/network.go

```diff
@@ -238,6 +241,11 @@ func newNetwork(
 		}
 	}
 
+	net.rootDir, err = os.MkdirTemp("", "avalanchego-network-runner-*")
```

Comment:
```suggestion
	net.rootDir, err = os.MkdirTemp("", "avalanche-network-runner-*")
```
to be consistent with namind